homeassistant/components/roku.py

```diff
@@ -0,0 +1,119 @@
+"""
+Support for Roku platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/roku/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.discovery import SERVICE_ROKU
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['python-roku==3.1.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'roku'
+
+SERVICE_SCAN = 'roku_scan'
+
+ATTR_ROKU = 'roku'
+
+DATA_ROKU = 'data_roku'
+DATA_ENTITIES = 'data_roku_entities'
```

Comment:
Define this in the platform. Stored entities should not be accessed outside of the platform. Not sharing the key will make this clearer.